javascript-modules-engine/tests/jahia-module/README.md

```diff
@@ -1,159 +1,14 @@
-<!--
-    Template for Readmes, see alternatives/examples here: https://github.com/matiassingers/awesome-readme
--->
+# jahia-module
 
-<!--
-    Badges provides a quick glance at the state of the repository and pointers to external resources.
-    More can be generated from here: https://shields.io/
--->
+There is a Jahia-wide convention: if `tests/` contains a `jahia-module/` directory, it is automatically provisioned to the test Jahia instance. An example of this convention can be found [in the `jahia-cypress` repository](https://github.com/Jahia/jahia-cypress/blob/41f0f5ccdc7228292ec4c72fb758f7f067314486/ci.build.sh#L22).
```

Comment:
let's remove the links to the `ci.build.sh` file as it's not really used anymore and would add confusion

Reply:
done for this one